packages/react-test-renderer/__tests__/shallow-test.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @emails react-core
+ */
+
+'use strict';
+
+describe('shallow', () => {
+  it('throws an error on import', () => {
+    expect(() => {
+      require('../shallow.js');
+    }).toThrow(
+      'react-test-renderer is deprecated. To use react-test-renderer/shallow, import react-shallow-renderer directly. See https://react.dev/warnings/react-test-renderer'
```

Comment:
https://react.dev/warnings/react-test-renderer is not talking about the shallow entrypoint at the moment. Is there a PR up fixing that?

Reply:
@eps1lon see https://github.com/reactjs/react.dev/pull/6689